Automattic/Gravatar-SDK-Android
# Remove ExpandableText and replace it uses with a Text composable

Closes #113 

### Description

Based on discussion we don't want to expand any of the information inside the profile components, so we no longer need the ExpandableText. If you want to see more, you should tap the View More button.

| Before | After |
|----------|-----------|
|<img width="250" alt="image" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/6974554/7e37c4fc-1e5d-4a94-ae69-359f80023cf6">|<img width="250" alt="Alt Text" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/6974554/77dee0c4-6527-460d-a0e0-7de84e1dec19">|

### Testing Steps

- Launch the demo app and verify everything looks like expected.
- Verify that tapping on an overflowed `AboutMe`, `Location` or `UserInfo` don't do anything.
